meilidb-http/src/routes/search.rs

```diff
@@ -155,13 +155,7 @@ pub async fn search_multi_index(mut ctx: Context<Data>) -> SResult<Response> {
 
     for index in index_list.clone() {
         if index == "*" {
-            index_list = ctx
-                .state()
-                .db
-                .indexes_names()
-                .map_err(ResponseError::internal)?
-                .into_iter()
-                .collect();
+            index_list = ctx.state().db.indexes_uids().into_iter().collect();
```

Comment:
Are you collecting a `Vec<String>` and store it inside of a `Vec<String>` ?
```suggestion
            index_list = ctx.state().db.indexes_uids();
```

Reply:
I need to convert it into an HashSet<String>